commands/version/README.md

```diff
@@ -51,7 +51,7 @@ Running `lerna version --conventional-commits` without the above flags will rele
 - [`--exact`](#--exact)
 - [`--force-publish`](#--force-publish)
 - [`--git-remote`](#--git-remote-name)
-- [`--github-release`](#--github-release)
+- [`--create-release`](#--create-release-type)
```

Comment:
This would force a 4.x wouldn't it as it's changing the argument entirely. Would we want to keep the old one and add the new one and deprecate later in a 4.x with more changes?

Reply:
This is correct. The old option can be used but it is deprecated.

https://github.com/lerna/lerna/pull/2073/files#diff-9577e51e2f56e073be1dcdf9bf503198R214

and

https://github.com/lerna/lerna/pull/2073/files#diff-0450c85a73670cf460e9ce9c771f100fR9

Thanks for have a look :)